prime-router/src/main/kotlin/Element.kt

```diff
@@ -1,5 +1,7 @@
 package gov.cdc.prime.router
 
+import com.google.i18n.phonenumbers.PhoneNumberUtil
+import java.text.DecimalFormat
```

Comment:
Google's I18n phone number package helps with the wide variety of phone numbers.